internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -86,11 +86,11 @@ const (
 	profileDocURL     = "https://docs.mongodb.com/mongocli/stable/configure/#std-label-mcli-configure"
 )
 
-// DefaultRegions represents the regions available for each cloud service provider
+// DefaultRegions represents the regions available for each cloud service provider for the M2 tier
 var DefaultRegions = map[string][]string{
-	"AWS":   {"US_EAST_1", "US_WEST_2", "AP_SOUTH_1", "AP_EAST_2", "EU_WEST_1", "EU_CENTRAL_1", "ME_SOUTH_1", "AF_SOUTH_1"},
-	"GCP":   {"CENTRAL_US", "CANADA_CENTRAL", "WESTERN_EUROPE", "ASIA_SOUTH_EAST", "SOUTH_AFRICA_NORTH", "UAE_NORTH"},
-	"AZURE": {"US_EAST_2", "US_WEST", "EUROPE_NORTH"},
+	"AWS":   {"US_EAST_1"},
+	"GCP":   {"CENTRAL_US"},
+	"AZURE": {"US_EAST_2"},
```

Comment:
do we need to keep this? I assume we would remove it with the new API

Reply:
I kept this in case the API returns an error.

Reply:
let's discuss over the error comment